### ISSUE TITLE
Git friendly universe file format

### ISSUE THREAD

**Issue by sergeypdev** [nem0/LumixEngine#1405]
Hello @nem0, first of all I really like your engine, great job!

I was wondering if you have any plans to add an alternative text based file format for universes? I can't imagine how to solve merge conflicts in a binary file. Maybe there is some workaround?

Edit:
Maybe there could be an engine cli tool that can merge two universes?

**Comment by nem0** [comment]
Hi, I used to have a text based file format for universes, similar to http://the-witness.net/news/2011/12/engine-tech-concurrent-world-editing/ but I never really made use of its advantages, but constantly paid the price, so I dropped it. I don't plan to add anything like it again. 

There's an option to load several universe files into one universe, it's how prefabs work and I use it in my game from Lua API. This is not exposed in editor but I plan to do it. This way several people can work on different parts of one scene at the same time.

Note: This is what I call from loading screen to load my level
```
function start()
    LumixAPI.pause(true)
    this.universe:load("level", function()
        while LumixAPI.hasFilesystemWork() do
            LumixAPI.processFilesystemWork()
        end
        this.gui_rect.enabled = false
        LumixAPI.pause(false)
        this.universe:getScene("on_the_hunt"):ready()
    end)
end
```
